TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/ProfessionalStatus.cs

```diff
@@ -26,8 +26,7 @@ public ProfessionalStatus()
     public Country? TrainingCountry { get; }
     public required Guid? TrainingProviderId { get; set; }
     public TrainingProvider? TrainingProvider { get; }
-    public required Guid? InductionExemptionReasonId { get; set; }
-    public InductionExemptionReason? InductionExemptionReason { get; }
+    public required bool? ExemptFromInduction { get; set; }
```

Comment:
I made it nullable because it doesn't apply when the status isn't 'Approved' or 'Awarded', rather than it being false